### ISSUE TITLE
static assets issues

### ISSUE THREAD

**Issue by brianleroux** [architect/architect#427]
```bash
mkdir foo
cd foo
arc init
```

Edit to look like

```arc
@app
foo

@static
fingerprint true

@http
get /foo
```

- [ ] bug#1 and run `arc deploy` to see it deploy `index-asdf.html` which won't get found
- [ ] bug#2 `arc deploy static` throws an error

**Comment by ryanblock** [comment]
I'd say `bug#1 and run arc deploy to see it deploy index-asdf.html which won't get found` is not a bug, but an enhancement – and something already considered for future `arc.static()`: https://github.com/architect/functions/blob/master/src/static/index.js#L31-L34

Let's break that one out into a separate issue!

**Comment by ryanblock** [comment]
Fix for bug#2 released in `6.0.1`
